src/picongpu/include/fields/background/cellwiseOperation.hpp

```diff
@@ -38,7 +38,7 @@ namespace cellwiseOperation
 
     /** Kernel that calls T_OpFunctor and T_ValFunctor on each cell of a field
      *
-     *  Pseudo code: opFunctor( cell, valFunctor( globalCellIdx, currentStep ) );
+     *  Pseudo code: opFunctor( cell, valFunctor( totalCellIdx, currentStep ) );
```

Comment:
I don't think so...? I implemented it so it can be used with a total position for background fields.